index.rst

```diff
@@ -90,19 +90,21 @@ contributing to Python:
 Status of Python branches
 -------------------------
 
-+------------------+--------------+-------------+----------------+----------------+-------------------+
-| Branch           | Schedule     | Status      | First release  | End-of-life    | Release manager   |
-+==================+==============+=============+================+================+===================+
-| master           | :pep:`596`   | features    | *TBD*          | *TBD*          | Łukasz Langa      |
-+------------------+--------------+-------------+----------------+----------------+-------------------+
-| 3.8              | :pep:`569`   | bugfix      | 2019-10-14     | *2024-10*      | Łukasz Langa      |
-+------------------+--------------+-------------+----------------+----------------+-------------------+
-| 3.7              | :pep:`537`   | bugfix      | 2018-06-27     | *2023-06-27*   | Ned Deily         |
-+------------------+--------------+-------------+----------------+----------------+-------------------+
-| 3.6              | :pep:`494`   | security    | 2016-12-23     | *2021-12-23*   | Ned Deily         |
-+------------------+--------------+-------------+----------------+----------------+-------------------+
-| 3.5              | :pep:`478`   | security    | 2015-09-13     | *2020-09-13*   | Larry Hastings    |
-+------------------+--------------+-------------+----------------+----------------+-------------------+
++------------------+--------------+-------------+----------------+----------------+-----------------------+
+| Branch           | Schedule     | Status      | First release  | End-of-life    | Release manager       |
++==================+==============+=============+================+================+=======================+
+| master           | *TBD*        | features    | *TBD*          | *TBD*          | Pablo Galindo Salgado |
++------------------+--------------+-------------+----------------+----------------+-----------------------+
+| 3.9              | :pep:`596`   | bugfix      | *TBD*          | *TBD*          | Łukasz Langa          |
```

Comment:
First release can be given as `*2020-10*` (maybe even `*2020-10-05*`, since it is specified in the PEP) here.